e2e/test/scenarios/embedding/sdk-iframe-embedding-setup/get-code.cy.spec.ts

```diff
@@ -127,13 +134,14 @@ H.describeWithSnowplow(suiteTitle, () => {
 
     getEmbedSidebar().within(() => {
       cy.findByLabelText("Single sign-on (SSO)").click();
+      codeBlock().should("not.contain", "useExistingUserSession");
+
+      cy.findByText(/Copy Code/).click();
 
       H.expectUnstructuredSnowplowEvent({
-        event: "embed_wizard_auth_selected",
-        event_detail: "sso",
+        event: "embed_wizard_code_copied",
+        event_detail: "auth=sso",
```

Comment:
When tracking for copied code, also report whether the copied code is SSO or user session.